### PR TITLE
UI for `neo-express claim` dialog

### PR DESCRIPTION
**Pull request by djnicholson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -158,30 +168,34 @@
         "vscode:prepublish": "npm run compile",
         "compile": "npm install && npm run compile-ts && npm run compile-css && npm run compile-js",
         "compile-ts": "tsc -p ./",
-        "compile-css": "npm run compile-css-1 && npm run compile-css-2 && npm run compile-css-3 && npm run compile-css-4 && npm run compile-css-5",
+        "compile-css": "npm run compile-css-1 && npm run compile-css-2 && npm run compile-css-3 && npm run compile-css-4 && npm run compile-css-5 && npm run compile-css-6",
```
Comment (devhawk) [inline package.json]: There isn't a way simpler way to compile all the .scss files?

File: package.json [hunk 2]
```diff
@@ -158,30 +168,34 @@
...
         "compile-css-1": "node-sass src/panels/tracker.scss -o out/panels",
         "compile-css-2": "node-sass src/panels/invoke.scss -o out/panels",
         "compile-css-3": "node-sass src/panels/create.scss -o out/panels",
         "compile-css-4": "node-sass src/panels/newwallet.scss -o out/panels",
         "compile-css-5": "node-sass src/panels/transfer.scss -o out/panels",
-        "compile-js": "npm run compile-js-1 && npm run compile-js-2 && npm run compile-js-3 && npm run compile-js-4 && npm run compile-js-5",
+        "compile-css-6": "node-sass src/panels/claim.scss -o out/panels",
+        "compile-js": "npm run compile-js-1 && npm run compile-js-2 && npm run compile-js-3 && npm run compile-js-4 && npm run compile-js-5 && npm run compile-js-6",
```
Comment (devhawk) [inline package.json]: Or a simpler way to compile these JS files?

File: package.json (hunk 1 above)
Comment (djnicholson) [reply package.json]: Yes, I can write a short script to enumerate the files that need compiling. Tracking with https://github.com/neo-project/neo-visual-tracker/issues/25

File: package.json (hunk 2 above)
Comment (djnicholson) [reply package.json]: Yes, I can write a short script to enumerate the files that need compiling. Tracking with https://github.com/neo-project/neo-visual-tracker/issues/25

**Pull request merged**